package.json

```diff
@@ -65,7 +65,7 @@
     "lint:ec": "eclint check && eclint check $npm_package_config_dotfiles",
     "lint:js": "eslint $npm_package_config_js",
     "lint": "npm run lint:ec && npm run lint:js",
-    "serve": "serve dist --port $npm_package_config_port --cache 0",
+    "serve": "serve dist -l $npm_package_config_port",
```

Comment:
`-l` -> `--local`

prefer long format flags in scripts

Reply:
Resolved in https://github.com/shower/core/pull/134/commits/345f6ded4e79d05735d2b344bcb7c76a511a1e41